model/finance.php

```diff
@@ -460,4 +461,28 @@ function zustatek() {
     return $this->zustatek;
   }
 
+  /**
+   * Zjistí datum posledního provedení platby
+   */
+  private function posledniPlatba(){
```

Comment:
Rozdělení na dvě metody tady nemá moc přidanou hodnotu, lépe (přehledněji) udělat to lazy loadingem, tedy zhruba následujícím stylem:

```
function hustota() {
  if(!isset($this->hustota)) {
    $this->hustota = $this->hmotnost / $this->objem;
    // případně nějaký složitější výpočet
  }
  return $this->hustota;
}